src/main/java/com/amplitude/BatchHttpCall.java

```diff
@@ -1,10 +1,11 @@
 package com.amplitude;
 
 public class BatchHttpCall extends HttpCall {
-  private static String apiUrl = Constants.BATCH_API_URL;
+  private static String apiUrl;
```

Comment:
why we make this static earlier? I don't see we have a strong need for it.

Reply:
+1